frontend/packages/dev-console/src/components/project-access/RenderProjectAccessPage.tsx

```diff
@@ -2,14 +2,14 @@ import * as React from 'react';
 import { Firehose } from '@console/internal/components/utils';
 import { withStartGuide } from '@console/internal/components/start-guide';
 import ProjectListPage from '../projects/ProjectListPage';
-import ProjectAccess, { ProjectAccessProps } from './ProjectAccess';
+import ProjectAccess from './ProjectAccess';
 
 export interface RenderProjectAccessPageProps {
   namespace: string;
 }
 
-const RenderProjectAccessPage: React.FC<RenderProjectAccessPageProps> = ({ namespace }) => {
-  const props: ProjectAccessProps = {
+export const RenderProjectAccessPage: React.FC<RenderProjectAccessPageProps> = ({ namespace }) => {
+  const props: React.ComponentProps<typeof ProjectAccess> = {
     formName: 'project access',
     namespace,
   };
```

Comment:
This variable creation is a bit bizarre in itself... was there any reason why you separated this into a variable instead of directly on the component? (originally when you did the feature - no change needed now, just curious)

Reply:
@andrewballantyne  yes, I could have directly passed formName and namespace. It might have gone unnoticed when I did the feature